tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -2947,6 +2957,74 @@ def _sanitize_stack_args(self, num_bag_folds, num_bag_sets, num_stack_levels, ti
             raise ValueError(f'num_bag_sets must be an integer. (num_bag_sets={num_bag_sets})')
         return num_bag_folds, num_bag_sets, num_stack_levels
 
+    def interpretable_models_summary(self, verbosity=0):
+        '''Summary of fitted interpretable models along with their corresponding complexities
+        '''
+        d = self.fit_summary(verbosity=verbosity)
```

Comment:
Its a bit strange that this is very similar to leaderboard but has different names for the same columns in a different order. Why not just call predictor.leaderboard() internally and add a couple new columns to it? In general, I'd recommend using leaderboard rather than fit_summary.